src/AbstractServer.php

```diff
@@ -6,17 +6,17 @@
  * @license   https://github.com/laminas/laminas-server/blob/master/LICENSE.md New BSD License
  */
 
+declare(strict_types=1);
+
 namespace Laminas\Server;
 
 use ReflectionClass;
+use ReflectionException;
 
-/**
- * Abstract Server implementation
- */
 abstract class AbstractServer implements Server
 {
     /**
-     * @var bool Flag; whether or not overwriting existing methods is allowed
+     * @var bool
      */
     protected $overwriteExistingMethods = false;
```

Comment:
Which PHP version are you targeting for this update? If you were to bump to 7.4, we could do property typehints here, which would make many of these property docblocks completely unnecessary!

Reply:
Absolutely right, typed property would bring some welcomed improvements. Currently `dev-3.0` is bumped to 7.2. Would a jump to 7.4 be generally be accepted?

Reply:
Yes! Particularly as there are definitely features that you could use here.

Reply:
Ok. Then my suggestion would be: 1) merging this PR, 2) bumping to PHP 7.4 and 3) then adding typed properties.

Reply:
@weierophinney Are you okay with me resolving this point? 😊

Reply:
Yep - and I'm doing it now. :smile: